app/src/test/java/org/simple/clinic/forgotpin/createnewpin/ForgotPinCreateNewPinScreenControllerTest.kt

```diff
@@ -121,5 +123,7 @@ class ForgotPinCreateNewPinScreenControllerTest {
 
     controllerSubscription = uiEvents.compose(controller)
         .subscribe { it.invoke(screen) }
+
+    uiEvents.onNext(ScreenCreated())
```

Comment:
Before we weren't triggering `ScreenCreated` in every test, why are we doing it now?

Reply:
We want to simulate the `Init` functionality. Since `Init` is going to be triggered for every test, we will move trigger `ScreenCreated` event on every test. Eventually we will replace it once the migration is completed and we remove the controller from tests.